### ISSUE TITLE
Use Mean-Square Error as objective

### ISSUE THREAD

**Issue by boom85423** [jiaxiang-cheng/PyTorch-CNN-for-RUL-Prediction#2]
I wonder why you use cross-entropy loss rather than mean-square loss for a regression problem? Have you compared the difference? thanks a lot !

**Comment by jiaxiang-cheng** [comment]
> I wonder why you use cross-entropy loss rather than mean-square loss for a regression problem? Have you compared the difference? thanks a lot !

Thanks for bringing up this point. I indeed tested both ways to check the performance. Seems cross-entropy loss works better with this model, and accordingly the outputs will be different cuz in this case it will be a classification problem rather than a regression problem. If you are interested you can simply change the output layer with a scalar then train with mse loss.
